# [Bug] say something

`Need Reproduce`

<!--
感谢您向我们反馈问题，为了高效的解决问题，我们期望你能提供以下信息：
-->

## What happens?

import {connect} from 'umi';

<!-- 清晰的描述下遇到的问题。-->

导入 connect报错

## Mini Showcase Repository(REQUIRED)

<img width="908" alt="image" src="https://user-images.githubusercontent.com/18204322/223722951-3526b611-8c80-4ffa-9df0-b4ba447eae93.png">

## fz6m

umi 默认没有集成 dva ，需要从 `@umijs/max` 导入（如果你用了 max ）。

## sanfire86626

import {connect} from 'umi'import {connect} from **_@_**.\*\*\*/max'
都不行啊，麻烦解决下吧

------------------&nbsp;原始邮件&nbsp;------------------
发件人: **_@_**.**_&gt;;
发送时间: 2023年3月8日(星期三) 晚上9:58
收件人: _**@**_._**&gt;;
抄送: **_@_**.**_&gt;; _**@**_._**&gt;;
主题: Re: [umijs/umi] [Bug] say something (Issue #10706)

umi 默认没有集成 dva ，需要从 @umijs/max 导入（如果你用了 max ）。

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: **_@_**.\*\*\*&gt;

## fz6m

需要给最小复现。
